02_Util/src/server/example/server.ts

```diff
@@ -0,0 +1,702 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+/* eslint-disable */
+
```

Comment:
Yeah I think previously this was all in Base before, and I believe I moved it here because to build things properly because of the dependencies not being available where they needed to be, I moved it to Util as it made more sense. At some point Thana made this PR that moved this into Util - `https://github.com/citrineos/citrineos-core/pull/56` and this code is now here - https://github.com/citrineos/citrineos-core/blob/rc-1.1.0/02_Util/src/networkconnection/WebsocketNetworkConnection.ts - I will remove these files from this PR